tests/ui/traits/const-traits/const_closure-const_trait_impl-ice-113381.rs

```diff
@@ -13,6 +13,4 @@ impl Foo for () {
 
 fn main() {
     (const || (()).foo())();
-    // ^ ERROR: cannot call non-const method `<() as Foo>::foo` in constant functions
```

Comment:
We sometimes leave these in to keep some context for when someone fixes the test. Fine here tho